gridscale/provider.go

```diff
@@ -1,12 +1,15 @@
 package gridscale
 
 import (
+	"runtime"
 	"strings"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
+const version = "1.8.1"
```

Comment:
Hmm, yeah. Might work for now. But we can get that thing from goreleaser I thing. Right?

```
builds:
    ldflags:
      - "-s -w -X github.com/gridscale/terraform-provider-gridscale/somePackage.GitCommit={{.Commit}} -X github.com/gridscale/terraform-provider-gridscale/somePackage.Version={{.Version}}"
```

This way it always works including "dirty" builds.